app/containers/MessageBox/buttons/OpenFullScreenButton.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import BaseButton from './BaseButton';
+
+const OpenFullScreenButton = React.memo(({ theme, onPress }) => (
+	<BaseButton
+		onPress={onPress}
+		testID='messagebox-full-screen'
```

Comment:
In general you want to avoid things like `testID` and test more from the users point of view e.g. find by `accessibilityLabel` instead.

Reply:
Right. I just use this props because I want to use the `BaseButton` and this props is required here https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/9dbe10bcf871392ed09b512152bc921578150e9f/app/containers/MessageBox/buttons/BaseButton.js#L27